README.md

```diff
@@ -1 +1,42 @@
-# Hello-World-Project
\ No newline at end of file
+# Hello-World
+This Project is created purposely to understand the basis of Git and GitHub flow
+
+## Built With
+* VsCode
+* HTML
+* CSS
+
+## Requirements
+* Create a new GitHub Repo and a new branch with GitHub flow
+*  Setup up Linter for HTML and CSS
+
+## Getting Started
+This project was setup using Visual Studio Code Editor. A linter was as well setup for the HTML and CSS files.
+Irrelevant files were specified in .gitignore file including the test file.
+
+## Prerequisite
+How to use a Computer
+
+## Setup
+
+## Install
+
+## Usage
+
+## Run tests
+
+## Deployment
+
+## Author
+* GitHub: @OyePriscilla
```

Comment:
[OPTIONAL] It is better to add your GitHub link like this `Github: [OyePriscilla](https://github.com/OyePriscilla)` instead of writing your username like `GitHub: @OyePriscilla`